Synthesis failed for python-trace

Hello! Autosynth couldn't regenerate python-trace. :broken_heart:

Here's the output from running `synth.py`:

```
proto.
2020-07-01 08:41:20,392 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpdmd9a2cs.
2020-07-01 08:41:20,399 synthtool [INFO] > Replaced 'from google.cloud.devtools.cloudtrace_v1.proto import ' in google/cloud/trace_v1/proto/trace_pb2_grpc.py.
2020-07-01 08:41:20,403 synthtool [INFO] > Replaced 'google-cloud-devtools-cloudtrace' in google/cloud/trace_v1/gapic/trace_service_client.py.
2020-07-01 08:41:20,403 synthtool [DEBUG] > Generating code for: //google/devtools/cloudtrace/v2:devtools-cloudtrace-v2-py.
2020-07-01 08:41:22,128 synthtool [DEBUG] > Copy: google/devtools/cloudtrace/v2/trace.proto to /tmpfs/tmp/tmpa3qmpy4g/google/cloud/trace_v2/proto/trace.proto
2020-07-01 08:41:22,129 synthtool [DEBUG] > Copy: google/devtools/cloudtrace/v2/tracing.proto to /tmpfs/tmp/tmpa3qmpy4g/google/cloud/trace_v2/proto/tracing.proto
2020-07-01 08:41:22,129 synthtool [SUCCESS] > Placed proto files into /tmpfs/tmp/tmpa3qmpy4g/google/cloud/trace_v2/proto.
2020-07-01 08:41:22,129 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpa3qmpy4g.
2020-07-01 08:41:22,137 synthtool [INFO] > Replaced 'from google.cloud.devtools.cloudtrace_v2.proto import ' in google/cloud/trace_v2/proto/tracing_pb2.py.
2020-07-01 08:41:22,138 synthtool [INFO] > Replaced 'from google.cloud.devtools.cloudtrace_v2.proto import ' in google/cloud/trace_v2/proto/tracing_pb2_grpc.py.
2020-07-01 08:41:22,139 synthtool [INFO] > Replaced 'google-cloud-devtools-cloudtrace' in google/cloud/trace_v2/gapic/trace_service_client.py.
.coveragerc
.flake8
.github/CONTRIBUTING.md
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.gitignore
.kokoro/build.sh
.kokoro/continuous/common.cfg
.kokoro/continuous/continuous.cfg
.kokoro/docs/common.cfg
.kokoro/docs/docs.cfg
.kokoro/presubmit/common.cfg
.kokoro/presubmit/presubmit.cfg
.kokoro/publish-docs.sh
.kokoro/release.sh
.kokoro/release/common.cfg
.kokoro/release/release.cfg
.kokoro/samples/lint/common.cfg
.kokoro/samples/lint/continuous.cfg
.kokoro/samples/lint/periodic.cfg
.kokoro/samples/lint/presubmit.cfg
.kokoro/samples/python3.6/common.cfg
.kokoro/samples/python3.6/continuous.cfg
.kokoro/samples/python3.6/periodic.cfg
.kokoro/samples/python3.6/presubmit.cfg
.kokoro/samples/python3.7/common.cfg
.kokoro/samples/python3.7/continuous.cfg
.kokoro/samples/python3.7/periodic.cfg
.kokoro/samples/python3.7/presubmit.cfg
.kokoro/samples/python3.8/common.cfg
.kokoro/samples/python3.8/continuous.cfg
.kokoro/samples/python3.8/periodic.cfg
.kokoro/samples/python3.8/presubmit.cfg
.kokoro/test-samples.sh
.kokoro/trampoline.sh
CODE_OF_CONDUCT.md
CONTRIBUTING.rst
LICENSE
MANIFEST.in
docs/_static/custom.css
docs/_templates/layout.html
docs/conf.py.j2
docs/multiprocessing.rst
noxfile.py.j2
renovate.json
Skipping: samples/AUTHORING_GUIDE.md
Skipping: samples/CONTRIBUTING.md
scripts/decrypt-secrets.sh
scripts/readme-gen/readme_gen.py.j2
scripts/readme-gen/templates/README.tmpl.rst
scripts/readme-gen/templates/auth.tmpl.rst
scripts/readme-gen/templates/auth_api_key.tmpl.rst
scripts/readme-gen/templates/install_deps.tmpl.rst
scripts/readme-gen/templates/install_portaudio.tmpl.rst
setup.cfg
testing/.gitignore
2020-07-01 08:41:22,256 synthtool [INFO] > Replaced '[]sphinx[]' in noxfile.py.
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/env/bin/nox", line 11, in <module>
    sys.exit(main())
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/nox/__main__.py", line 59, in main
    tasks.final_reduce,
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/nox/workflow.py", line 49, in execute
    return_value = function_(*args, global_config=global_config)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/nox/tasks.py", line 60, in load_nox_module
    "user_nox_module", global_config.noxfile
  File "<frozen importlib._bootstrap_external>", line 399, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 823, in load_module
  File "<frozen importlib._bootstrap_external>", line 682, in load_module
  File "<frozen importlib._bootstrap>", line 265, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-trace/noxfile.py", line 19
    from __future__ "sphinx<3.0.0"m"sphinx<3.0.0"ort ab"sphinx<3.0.0"olute_"sphinx<3.0.0"m"sphinx<3.0.0"ort
                                 ^
SyntaxError: invalid syntax
2020-07-01 08:41:22,395 synthtool [ERROR] > Failed executing nox -s blacken:

None
2020-07-01 08:41:22,396 synthtool [DEBUG] > Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/python-trace/synth.py", line 65, in <module>
    s.shell.run(["nox", "-s", "blacken"], hide_output=False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['nox', '-s', 'blacken']' returned non-zero exit status 1.
2020-07-01 08:41:22,438 autosynth [ERROR] > Synthesis failed
2020-07-01 08:41:22,438 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at dbee00d chore: update admonition for py2 (#25)
2020-07-01 08:41:22,449 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-01 08:41:22,455 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing google/__pycache__/
Removing google/cloud/__pycache__/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/4461171d-7395-48e4-9813-1e2e798ab425/targets/github%2Fsynthtool;config=default/tests;query=python-trace;failed=false).

--- a/synth.metadata
+++ b/synth.metadata
@@ -3,16 +3,16 @@
     {
       "git": {
         "name": ".",
-        "remote": "https://github.com/googleapis/python-trace.git",
-        "sha": "95ffd4e247b48d055bbe626f8231792bf0015d20"
+        "remote": "git@github.com:googleapis/python-trace",
+        "sha": "dbee00dcf67d988cf0ec4054619e74d80836800b"
       }
     },
     {
       "git": {
         "name": "googleapis",
         "remote": "https://github.com/googleapis/googleapis.git",
-        "sha": "b882b8e6bfcd708042ff00f7adc67ce750817dd0",
-        "internalRef": "318028816"
+        "sha": "a758393a3f77b11b0ddbf62d468454dd6492f48a",
+        "internalRef": "319296039"
       }
     },
     {

--- a/synth.py
+++ b/synth.py
@@ -60,6 +60,6 @@ templated_files = common.py_library(cov_level=98)
 s.move(templated_files)
 
 # TODO(busunkim): Use latest sphinx after microgenerator transition
-s.replace("noxfile.py", '['"]sphinx["']', '"sphinx<3.0.0"')
+s.replace("noxfile.py", '''['"]sphinx["']''', '"sphinx<3.0.0"')
 
 s.shell.run(["nox", "-s", "blacken"], hide_output=False)

